src/ui/pages/customers.page.ts

```diff
@@ -1,25 +1,41 @@
 import { BasePage } from "./Salesportal.BasePage.page";
 
-
 export class Customers extends BasePage {
-  addNewCustomer = this.page.locator('//*[@id="title"]/div[1]/button');
-  searchInput = this.page.getByPlaceholder("Type a value...");
-  searchButton = this.page.locator("#search-customer");
-  customerInCustomerList = this.page.locator("#table-customers > tbody > tr > td:nth-child(1)");
-  
-  
+  readonly addNewCustomer = this.page.locator('//*[@id="title"]/div[1]/button');
```

Comment:
Не стоит использовать `Copy xpath` 🙂
```suggestion
  readonly addNewCustomer = this.page.locator('button.page-title-button);
```